src/sonic-yang-mgmt/sonic_yang.py

```diff
@@ -0,0 +1,450 @@
+import yang as ly
+import sys
+import json
+import glob
+
+"""
+Yang schema and data tree python APIs based on libyang python
+"""
+class sonic_yang:
+    def __init__(self, yang_dir):
+        self.yang_dir = yang_dir
+        self.ctx = None
+        self.module = None
+        self.root = None
+
+        try:
+            self.ctx = ly.Context(yang_dir)
+        except Exception as e:
+            print(e)
+
+    """
+    load_schema_module(): load a Yang model file
+    input:    yang_file - full path of a Yang model file
+    returns:  True - success   False - failed
+    """
+    def load_schema_module(self, yang_file):
+        try:
+            self.ctx.parse_module_path(yang_file, ly.LYS_IN_YANG)
+        except Exception as e:
+            print(e)
```

Comment:
I think all prints, either should be meaningful or should be removed.

Reply:
Error messages added